src/main/java/PDLayer.mdl

```diff
@@ -604,6 +608,10 @@ type PDInkAnnot extends PDMarkupAnnot {
 type PDLinkAnnot extends PDAnnot {
 	% link to the destination of link annotation 
 	link Dest: PDDestination?;
+	% object key of the Link annotation that target a different location, but contains in the same structure element
```

Comment:
% object + generation number of the Link annotation that targets a different location, but is contained in the same structure element as another Link annotation